<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathPenaltyDialog</class>
 <widget class="QDialog" name="PathPenaltyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Penalties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="shapefileGroup">
     <property name="title">
      <string>Shapefiles</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Resistance to cross the area (i.e. penalty)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nb of days closed per month</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="nbOfDaysClosedPerMonth">
     <property name="maximum">
      <double>31.000000000000000</double>
     </property>
     <property name="value">
      <double>31.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QDoubleSpinBox" name="weight">
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidgetPeriods">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabPageMonths">
      <attribute name="title">
       <string>Closure Months</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPageMetiers">
      <attribute name="title">
       <string>AND Metiers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="enableMetiers">
         <property name="text">
          <string>Select Metiers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="metierGroup">
         <property name="title">
          <string>Metiers</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPageVesselsSizes">
      <attribute name="title">
       <string>AND Vessels sizes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QGroupBox" name="sizeGroup">
         <property name="title">
          <string>Sizes</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbSize0">
            <property name="text">
             <string>&lt; 15m</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="cbSize1">
            <property name="text">
             <string>15-18m</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="cbSize2">
            <property name="text">
             <string>18-24m</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbSize3">
            <property name="text">
             <string>24-40m</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="cbSize4">
            <property name="text">
             <string>&gt; 40m</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="enableSizes">
         <property name="text">
          <string>Select Sizes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPageNations">
      <attribute name="title">
       <string>AND Vessels Nations</string>
      </attribute>
      <widget class="QCheckBox" name="enableNations">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>487</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Nations</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="nationGroup">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>515</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Nations</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="2">
         <widget class="QCheckBox" name="cbNation1">
          <property name="text">
           <string>BEL</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="cbNation2">
          <property name="text">
           <string>BGR</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QCheckBox" name="cbNation3">
          <property name="text">
           <string>CYP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QCheckBox" name="cbNation4">
          <property name="text">
           <string>DEU</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QCheckBox" name="cbNation5">
          <property name="text">
           <string>DNK</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QCheckBox" name="cbNation6">
          <property name="text">
           <string>ESP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="QCheckBox" name="cbNation7">
          <property name="text">
           <string>EST</string>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QCheckBox" name="cbNation8">
          <property name="text">
           <string>FIN</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="cbNation9">
          <property name="text">
           <string>FRA</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="cbNation10">
          <property name="text">
           <string>GBR</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="cbNation11">
          <property name="text">
           <string>GRC</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCheckBox" name="cbNation12">
          <property name="text">
           <string>HRV</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QCheckBox" name="cbNation13">
          <property name="text">
           <string>IRL</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QCheckBox" name="cbNation14">
          <property name="text">
           <string>ITA</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QCheckBox" name="cbNation15">
          <property name="text">
           <string>LTU</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QCheckBox" name="cbNation16">
          <property name="text">
           <string>LVA</string>
          </property>
         </widget>
        </item>
        <item row="2" column="8">
         <widget class="QCheckBox" name="cbNation17">
          <property name="text">
           <string>MLT</string>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <widget class="QCheckBox" name="cbNation18">
          <property name="text">
           <string>NLD</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="cbNation19">
          <property name="text">
           <string>POL</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="cbNation20">
          <property name="text">
           <string>PRT</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="cbNation21">
          <property name="text">
           <string>ROU</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QCheckBox" name="cbNation22">
          <property name="text">
           <string>SVN</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QCheckBox" name="cbNation23">
          <property name="text">
           <string>SWE</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="cbNation0">
          <property name="text">
           <string>All nations</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="5" colspan="2">
         <widget class="QCheckBox" name="cbNation24">
          <property name="text">
           <string>OTHERS</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>weight</tabstop>
  <tabstop>ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>PathPenaltyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
